libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -431,6 +439,17 @@ void lua_scripts::run(void) {
             lua_gc(L, LUA_GCCOLLECT, 0);
 
         } else {
+            if (reload) {
```

Comment:
I'd probably but this below the if check below, as that adds more insight if debugging when all scripts seem to pause for 10 seconds, alternatively send out a status text here if debugging that says "Lua: Reloading scripts in 10 seconds due to an error" or something similiar.

Reply:
put the whole lot in the if and added status text